src/test/java/magic/cascade/MemoryAdapter.java

```diff
@@ -31,7 +31,7 @@
 
             @Override
             public CreateOperation<T> from(Stream<T> data, QueryOptions options) {
```

Comment:
It **is** a stub, needed only for a specific test (this is why it is in `src/test` )